### ISSUE TITLE
Error when starting projec

### ISSUE THREAD

**Issue by marcelfenerich** [ramsey/php-library-skeleton#42]
When first installing the project I get an error on `Ramsey\Dev\LibraryStarterKit\Wizard::start`.

The error is: `No arguments expected, got "starter-kit".`

I'm using Ubuntu 20.04 and php 7.4

![image](https://user-images.githubusercontent.com/7074574/161591836-a203935b-3465-4fe5-adc1-b881bea54bf4.png)

What I'm doing wrong?

**Comment by marcelfenerich** [comment]
I believe it is happening because the project is not compatible to composer 2.

**Comment by ramsey** [comment]
The project is compatible with Composer 2, but the version of symfony/console that's bundled with Composer is older than what the ramsey/devtools plugin uses, and so it ends up in a conflicted state. I'm having a related problem over here: https://github.com/ramsey/conventional-commits/pull/45

I need to figure out a solution that avoids using Composer's bundled symfony/console.

**Comment by goodevilgenius** [comment]
Any workarounds to this until you get this figured out? Just had the issue myself.

**Comment by simpleapps-io** [comment]
I'm running into this as well. Any current workaround?

**Comment by ramsey** [comment]
I don't know a workaround at the moment. I need to find some time to revisit this and rework a few of the internals so that this runs smoothly, without conflicting with the bundled composer libraries.

**Comment by Romaixn** [comment]
Hello, same error here using PHP 8.1.10 and composer 2.3.10 ! Also an error when running `composer starter-kit` :

```
PHP Fatal error:  Declaration of Ramsey\Dev\Tools\Composer\Command\BaseCommand::getApplication(): Symfony\Component\Console\Application must be compatible with Composer\Command\BaseCommand::getApplication(): Composer\Console\Application in /home/romain/Documents/PHP/captcha-wally/vendor/ramsey/devtools-lib/src/Composer/Command/BaseCommand.php on line 135

Fatal error: Declaration of Ramsey\Dev\Tools\Composer\Command\BaseCommand::getApplication(): Symfony\Component\Console\Application must be compatible with Composer\Command\BaseCommand::getApplication(): Composer\Console\Application in /home/romain/Documents/PHP/captcha-wally/vendor/ramsey/devtools-lib/src/Composer/Command/BaseCommand.php on line 135
```

I would love to contribute if I find a solution to the problem.

**Comment by ramsey** [comment]
I've just released version 3.5, which fixes this issue for PHP 8.1+ and Composer 2.5+.

Unfortunately, I was unable to make it work for PHP 7.4 and earlier versions of Composer.
